Tests/Functional/Command/ImportFullCommandTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+* This file is part of the ONGR package.
+*
+* (c) NFQ Technologies UAB <info@nfq.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace ONGR\MagentoConnectorBundle\Tests\Functional\Command;
+
+use ONGR\ConnectionsBundle\Command\ImportFullCommand;
+use ONGR\ElasticsearchBundle\ORM\Manager;
+use ONGR\ElasticsearchBundle\ORM\Repository;
+use ONGR\MagentoConnectorBundle\Tests\app\fixtures\ExpectedDocuments\ExpectedDocuments;
+use ONGR\ElasticsearchBundle\Test\ElasticsearchTestCase;
+use ONGR\MagentoConnectorBundle\Tests\Functional\TestBase;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * This class contains tests to find out if services are defined correctly.
+ */
+class ImportFullCommandTest extends ElasticsearchTestCase
+{
+    /**
+     * Check if a document is saved as expected after collecting data from providers.
+     */
+    public function testDefaultImportCommand()
+    {
+        TestBase::setUpBeforeClass();
```

Comment:
what? why? setupbefore class is called before creating this class already, why call it twice?